lib/sda/local.go

```diff
@@ -229,6 +229,23 @@ func (l *LocalTest) GetPrivate(h *Host) abstract.Secret {
 	return h.private
 }
 
+// GetServices returns a slice of all services asked for.
+// The sid is the id of the service that will be collected.
+func (l *LocalTest) GetServices(hosts []*Host, sid ServiceID) []Service {
+	services := make([]Service, len(hosts))
+	for i, h := range hosts {
+		services[i] = l.Services[h.Entity.ID][sid]
+	}
+	return services
+}
+
+// MakeHELS is an abbreviation to make a Host, an EntityList, and a service
+func (l *LocalTest) MakeHELS(nbr int, sid ServiceID) ([]*Host, *EntityList, Service) {
```

Comment:
Do you use it somewhere ? It just appears on this PR but I can't see any use of it. 
And also, don't you think that'd more useful to have a function that returns the `Service` from the `Host` (or from `LocalTest`) BY Host ? And not just returning the `Service` of the **first** host ? Like 
```
func (l *LocalTest) Service(h *Host,sid *ServiceID) Service
```
Maybe not, I'm not sure, it's depending on the test you've made I guess.  know that in the `service_test.go`, I often needed to have control over the two services (in a 2-hosts setup). 
Orthogonal to that, we could add a comment saying that it returns the `Service` of the first `Host` == root.